components/browser_kit.rst

```diff
@@ -8,13 +8,6 @@ The BrowserKit Component
     The BrowserKit component simulates the behavior of a web browser, allowing
     you to make requests, click on links and submit forms programmatically.
 
-.. note::
```

Comment:
Section doc already exists, we can drop note
https://symfony.com/doc/current/components/browser_kit.html#making-external-http-requests